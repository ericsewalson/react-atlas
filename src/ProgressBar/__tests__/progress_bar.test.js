import React from "react";
import ProgressBar from "../index";

import renderer from "react-test-renderer";

describe("Test correct render", () => {
  it("Test correct render", function() {
    const tree = renderer
      .create(<ProgressBar>ProgressBar</ProgressBar>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
