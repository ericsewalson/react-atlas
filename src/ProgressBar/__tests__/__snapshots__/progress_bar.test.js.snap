// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test correct render Test correct render 1`] = `
<div
  aria-valuemax={100}
  aria-valuemin={0}
  aria-valuenow={0}
  className="linear indeterminate"
  style={undefined}
>
  <div>
    <span
      className="buffer"
      data-ref="buffer"
      style={undefined}
    />
    <span
      className="value"
      data-ref="value"
      style={
        Object {
          "backgroundColor": undefined,
        }
      }
    />
  </div>
</div>
`;
